src/videojs.ads.js

```diff
@@ -455,7 +437,7 @@ var
       // Returns a boolean indicating if given player is in live mode.
       // Can be replaced when this is fixed: https://github.com/videojs/video.js/issues/3262
       isLive: function(player) {
-        if (player.duration() === Infinity) {
+        if (player.currentTime() > 0 && player.duration() === Infinity) {
```

Comment:
While it might only be wrong in a very small number of cases, this additional condition seems dangerous to me. I worry we'd get some very hard to reproduce bug reports.